src/electron/updater.js

```diff
@@ -0,0 +1,82 @@
+const { autoUpdater } = require("electron-updater");
+const { version } = require("../../package.json");
+const logger = require("electron-log");
+
+const setupConfig = () => {
+	autoUpdater.logger = logger;
+	autoUpdater.logger.transports.file.level = "info";
+	autoUpdater.autoDownload = false;
+	autoUpdater.currentVersion = version;
+	autoUpdater.updateConfigPath = "app-update.yml";
```

Comment:
This is causing error not found in production mode. E2E can be detected with `process.env.ELECTRON_IS_E2E`.

Reply:
Included `ELECTRON_IS_E2E` check to add `app-update.yml`. Seems to be working ok now in dev and e2e. Thanks!